internal/kibana/ingestmanager/client_packages.go

```diff
@@ -0,0 +1,46 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package ingestmanager
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/packages"
+)
+
+// InstallPackage installs the given package in Fleet.
+func (c *Client) InstallPackage(pkg packages.PackageManifest) ([]packages.Asset, error) {
```

Comment:
We may need to unify our approaches to Kibana "subclients" as it looks like we duplicate some code. With this approach: https://github.com/elastic/elastic-package/blob/master/internal/kibana/client.go I encapsulated `dashboards` API in a separate `.go` file which shared a Kibana client. Do you have any preferences here?

The reason why I insist on the other approach is that `dashboards` API is not included within the `fleet` API (btw. we should rename it to fleet) and at some point we need to unify approaches.

Let's discuss the final design, so that this client is added in the right place in the source tree and we can adjust the rest of code.

Reply:
Agreed. In the beginning all we had was the ingestmanager (now fleet) client. But now that we have a more general kibana client, we should unify them. I took at stab at this in #210.